### ISSUE TITLE
there are still opengl erers on some devices

### ISSUE THREAD

**Issue by wesleyvanneck** [SachinVin/citra_android#72]
**Describe the bug**

**To Reproduce**
there are still opengl erors on some devices
just open a game
**Expected behavior**

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: [e.g. Samsung galaxy s8]
 - OS: [e.g. Android8.1]
 - Version [e.g. 1.0.0]

**Logs**
Add logs here.

[citra_log.txt](https://github.com/SachinVin/citra_android/files/2543181/citra_log.txt)

**Comment by Jack-Rickwood** [comment]
All exynos devices are currently getting this error i think, but it doesnt stop you from booting games.

**Comment by wesleyvanneck** [comment]
@SachinVin those errors make performance worse

**Comment by SachinVin** [comment]
Performance is probably not affected by these errors, you can see in the later part of the log that these errors disappear(after screen rotation). I'm not sure why the errors happen only on mali gpus :/

**Comment by Jack-Rickwood** [comment]
@SachinVin yeah, nothing would display for me until i rotated the screen, but after that it worked perfectly
